sagemaker-debugger/pytorch_profiling/pt-resnet-profiling-multi-gpu-multi-node.ipynb

```diff
@@ -9,14 +9,37 @@
     "This notebook will walk you through creating a PyTorch training job with the SageMaker Debugger profiling feature enabled. It will create a multi GPU multi node training.\n"
```

Comment:
<p>To use the new Debugger profiling features, ensure that you have the latest versions of SageMaker and SMDebug SDKs installed. The following cell updates the libraries and restart the kernel to apply the updates to your working environment.</p><p> </p><p> </p><p>^ please change this throughout the notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done.